doozerlib/cli/images_streams.py

```diff
@@ -616,7 +616,7 @@ def connect_issue_with_pr(github_client: Github, pr_url: str, issue: str):
         pr_url: The URL of the PR to align with
         issue: JIRA issue key
     """
-    pr_info = pr_url.split('/')
+    pr_info = pr_url.split('/')  # e.g. ['https:', '', 'github.com', 'openshift', 'origin', 'pull', '12']
     source_repo = github_client.get_repo(f'{pr_info[-4]}/{pr_info[-3]}')
```

Comment:
Why using reverse indexing? Either way, I think we need a comment making it clear what fields we are selecting

Reply:
I don't want to complicate the process of extracting the relevant fields from the pr URL address to get the pr object

Reply:
What exactly is against using indexes 3 and 4, rather than -4 and -3?

Reply:
the function is updated to us pr object now